audiomentations/augmentations/transforms.py

```diff
@@ -193,32 +193,69 @@ class AddGaussianSNR(BaseWaveformTransform):
 
     supports_multichannel = True
 
-    def __init__(self, min_SNR=0.001, max_SNR=1.0, p=0.5):
+    def __init__(self, 
+                 min_snr_in_db=3,
+                 max_snr_in_db=30,
+                 min_SNR=None, 
+                 max_SNR=None,
+                 p=0.5, 
+                 snr_characteristics=None
+     ):
         """
-        :param min_SNR: Minimum signal-to-noise ratio
-        :param max_SNR: Maximum signal-to-noise ratio
+        :param min_snr_in_db: Minimum signal-to-noise ratio in db
+        :param max_snr_in_db: Maximum signal-to-noise ratio in db
+        :param min_SNR: Minimum signal-to-noise ratio (legacy)
+        :param max_SNR: Maximum signal-to-noise ratio (legacy)
         :param p: The probability of applying this transform
+        :snr_characteristics: Use legacy mode or not
         """
         super().__init__(p)
+        self.snr_characteristics = snr_characteristics
+        if self.snr_characteristics is None:
+            warnings.warn(
+                'Please set snr_characteristics to either "legacy" or "correct".'
+                ' The default behavior may change from "legacy" to "default" in a later release'
```

Comment:
"legacy" to _"correct"_

Reply:
I think that there's no need of separate `snr_characteristics` argument.